src/test/java/org/zalando/nakadi/repository/kafka/KafkaTopicRepositoryTest.java

```diff
@@ -362,6 +374,38 @@ public void whenValidateInvalidCommitCursorsThenException() throws NakadiExcepti
                 });
     }
 
+    @Test
+    public void whenKafkaPublishTimeoutThenCircuitIsOpened() throws Exception {
```

Comment:
Where are you checking if the circuit opened in this test? 

I expected something like an assertion with 20 failures due to timeout and `1000 - 20` failures due to open circuit.